### DISCUSSION TITLE
Help building the environment

### DISCUSSION THREAD

**Discussion by tpike3** [jupyterlite/jupyterlite#1454] [Q&A]
I am attempting to use Jupyterlite with a Xeus-Python Kernel for the introductory tutorial for Mesa-Geo (Agent Based Modelling in Python with GIS). The idea is this will provide users a zero start up cost way to explore the library. 

However, I cannot find the right set-up to get the environment to work.

Any thoughts or insights would be much appreciated, below are what I am seeing are the critical parts

[Jupyterlite build](https://mesa-geo-test.readthedocs.io/en/latest/lite/lab/)

[readthedocs to initiate][(https://mesa-geo-test.readthedocs.io/en/latest/lite/lab/](https://mesa-geo-test.readthedocs.io/en/latest/tutorials/overview.html#))

[build-environment](https://github.com/tpike3/mesa-geo/blob/newsphinx/build-environment.yml) to install GIS based libraries not support through mamba conda-forge etc in xeus-python 

[docs envirnment](https://github.com/tpike3/mesa-geo/blob/newsphinx/docs/environment.yml) 

I am just not sure what string to pull next. Thank you

**Comment by jtpio** [comment]
Thanks @tpike3 for providing the details.

When trying the deployment, there is the following error in the dev tools console:

```
TypeError: type 'List' is not subscriptable
```

![image](https://github.com/user-attachments/assets/487e8b63-67f4-44ad-8b2d-e9f02e31339e)

It's not clear whether it's related to one of the dependencies, as this seems to be happening at kernel startup.

**Reply by jtpio** [reply]
> It's not clear whether it's related to one of the dependencies, as this seems to be happening at kernel startup.

I just tried with a minimal deployment with `jupyterlite-xeus==0.2.0a0` and only `xeus-python` in the run environment, and it seems to be working fine:

![image](https://github.com/user-attachments/assets/a6510282-4f2d-4505-ab5d-9ceb351fa289)

**Reply by tpike3** [reply]
Thanks @jtpio! I will take look!
